app/browser/api/ledgerNotifications.js

```diff
@@ -230,7 +230,7 @@ const showEnabledNotifications = (state) => {
       showReviewPublishers(now + ledgerUtil.milliseconds.day)
     }
   } else if (now - bootStamp > 7 * ledgerUtil.milliseconds.day) {
-    if (noBackupKeys(state) && shouldShowNotificationBackupKeys()) {
+    if (noBackupKeys(state) && shouldShowNotificationBackupKeys() && hasFunds(state)) {
```

Comment:
@mrose17, during our coding session, we created hasFunds but the linter picked it up as unused. My assumption was the notification should be shown based on the original criteria and also (added above) if the user has actual funds in the wallet. Can you confirm this? Thanks.